src/DynamoPackages/PackageUploadBuilder.cs

```diff
@@ -52,7 +52,8 @@ public static PackageUploadRequestBody NewRequestBody(Package package)
 
             return new PackageUploadRequestBody(package.Name, package.VersionName, package.Description, package.Keywords, package.License, package.Contents, PackageManagerClient.PackageEngineName,
                                                          version, engineMetadata, package.Group, package.Dependencies,
-                                                         package.SiteUrl, package.RepositoryUrl, package.ContainsBinaries, package.NodeLibraries.Select(x => x.FullName));
+                                                         package.SiteUrl, package.RepositoryUrl, package.ContainsBinaries, 
+                                                         package.NodeLibraries.Select(x => x.FullName), new List<string>());
```

Comment:
I can add a TODO here but pretty much this will be worked on for the package publishing part